src/main/resources/com/mathworks/ci/UseMatlabVersionBuildWrapper/help.html

```diff
@@ -1,3 +1,9 @@
-<p>
-Specify the MATLAB version for this build.
-</p>
\ No newline at end of file
+<div>
+    <p>
+        Use this list to specify the MATLAB version for the build. The plugin prepends your specified MATLAB to the PATH environment variable and uses it to perform the build. <br>
+        The list contains the names of all MATLAB versions that have been registered as a tool in <u>Jenkins Global Tool Configuration</u>. Select one of the tools, or select <b>Custom…</b> if you want to specify a different version. When you select <b>Custom…</b>, you must enter the full path to the desired MATLAB root folder in the <b>MATLAB root</b> box.

```

Comment:
The list contains --> The list includes
in <u>Jenkins Global Tool Configuration</u> --> in Jenkins

Reply:
Agreed, I had added the 'Global Tool Configuration' as  a way of letting user know where MATLAB versions are configured. But perhaps this information is already exposed in out plugin documentation.